matmul/matmul.rs/src/main.rs

```diff
@@ -58,10 +58,7 @@ fn main() {
 
     let left = calc(101);
     let right = -18.67;
-    if (left - right).abs() > 0.1 {
-        eprintln!("{left} != {right}");
-        process::exit(-1);
-    }
+    assert!((left - right).abs() <= 0.1);
```

Comment:
I'd prefer that the exact values would be shown here, e.g.

```rs
assert!((left - right).abs() <= 0.1, "{left} != {right}");
```

The same for other float point comparisons. Using the special macro is fine too, like mentioned in  https://stackoverflow.com/questions/30856285/assert-eq-with-floating-point-numbers-and-delta